rsts/deployment/deployment/multicluster.rst

```diff
@@ -133,12 +132,12 @@ file named ``secrets.yaml`` that looks like:
      namespace: flyte
    type: Opaque
    data:
-     cluster_1_token: {{ cluster 1 token here }}
-     cluster_1_cacert: {{ cluster 1 cacert here }}
-     cluster_2_token: {{ cluster 2 token here }}
-     cluster_2_cacert: {{ cluster 2 cacert here }}
-     cluster_3_token: {{ cluster 3 token here }}
-     cluster_3_cacert: {{ cluster 3 cacert here }}
+     cluster_1_token: "cluster-1-token-here"
```

Comment:
Are we assuming that we have 3 dataplanes or 2? Since in the first case we only need 2 entries. I'd also specify how to change the names of the clusters